gen_template_docs.py

```diff
@@ -148,9 +148,7 @@ def createTemplate(data, path):
                     if len(secretName) > 0:
  	                  tdata['objects'][0]['secrets'] = [{ "secretName": secretName[0], "secretFile": secretName[0] + ".json" }]
 		    
-        # currently the clustering section applies only to EAP templates
-        if re.match('^eap', path):
```

Comment:
Is this not needed because now the entire file is scoped to the EAP image?

Reply:
Correct. The script originated in the application-templates repo, so was filtering based on image names etc.